djangobaseview/static/js/guestbook/view/Greeting.js

```diff
@@ -0,0 +1,96 @@
+define([
+	"dojo/_base/declare",
+	"dojo/_base/fx",
+	"dojo/_base/lang",
+	"dojo/on",
+	"dojo/query",
+	"dojo/dom-style",
+	"dojo/dom",
+	"dojo/text!./templates/Greeting.html",
+	"dijit/registry",
+	"dijit/form/Button",
+	"dijit/form/Form",
+	"dijit/form/TextBox",
+	"./_ViewBaseMixin"
+
+], function(declare, baseFx, lang, on, query, domStyle, dom, template,
+			registry, button, form, textbox, _ViewBaseMixin){
+
+		return declare("WidgetGreeting", [_ViewBaseMixin], {
+
+			templateString: template,
+			guestbookStore: null,
+			guestbookName: "",
+			greeting_id: "",
```

Comment:
các thuộc tính của greeting hãy gom vào this.model 
- tạo models js/guestbook/models/Greeting.js